client_code/ProgressBar/Determinate/__init__.py

```diff
@@ -33,6 +35,24 @@ def progress(self, value):
         self._props["progress"] = value
         self.indicator_dom_node.style.setProperty("width", f"{value:%}")
 
+    @property
+    def height(self):
+        return self._height
+
+    @height.setter
+    def height(self, value):
+        self._height = value
+        self.indicator_dom_node.style.setProperty("height", _css_length(value or "3px"))
+
+    @property
+    def width(self):
+        return self._width
+
+    @width.setter
+    def width(self, value):
+        self._width = value
+        self.dom_node.style.setProperty("width", _css_length(value or "3px"))
```

Comment:
Let's fix the default widths 😊

Reply:
In what way do you mean? moving the or statement outside the function call, adding a default constant variable or readding it to the css?

Reply:
Oh right, that one should be 100%, my bad. I'll do that tomorrow, already shut down the computer for the night, haha.

Reply:
Just di that in the latest push, lmk if there's anything else :)